custom-instrument-apps/instrument-webframework/README.md

```diff
@@ -0,0 +1,57 @@
+# Example instrumentation of a web framework application
+This is an example app that uses the `newrelic.instrumentWebframework` [API function](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api/#instrumentWebframework) and associated [WebFramework shim API](https://newrelic.github.io/node-newrelic/WebFrameworkShim.html) to instrument a hypothetical web framework. This sample instrumentation is an similar implementation of the [New Relic web framework instrumentation tutorial](https://newrelic.github.io/node-newrelic/tutorial-Webframework-Simple.html).
+
+
+## Getting started
+
+1. Clone or fork, this repository,
+
+2. Go into this subdirectory,
+
+``` sh
+cd newrelic-node-examples/custom-instrument-apps/instrument-webframework
+```
+
+3. Install dependencies and run application
+
+```sh
+npm ci
+NEW_RELIC_LICENSE_KEY=<Your New Relic License Key> npm start
```

Comment:
Personally, I would avoid suggesting that folks write their key to their shell history. It looks like the included `newrelic.js` has the same "replace me" string in it. So I'd insert a step before this one to prompt the reader to update this config first.

Reply:
I prefer putting the license key in the config file too. This has been updated.